doc/compiling/windows.md

```diff
@@ -1,69 +1,129 @@
+
+Install from [cmake.org/download](https://cmake.org/download/). Once installed, you should be able to run `cmake-gui` from the start menu.
+
+Installing CMake this way also adds it to your PATH environment variable, and you should be able to run `(Get-Command cmake).Path` in PowerShell or `which cmake` in Bash and get a result.
+
+If you don't install CMake manually, and are relying on the version that comes bundled with Visual Studio, you can follow these steps:
+
+1. Find the `/path/to/bin/CMake.exe`. It should be at or around `C:\Program Files\Microsoft Visual Studio\2022\Community\Common7\IDE\CommonExtensions\Microsoft\CMake\CMake\bin\cmake.exe`.
+1. Edit your PATH environment variable to include the `/path/to/bin` (but not `cmake.exe`).
+
+    1. Start menu > Search "env" > Edit the system environment variables > Environment Variables... > Path > Edit... > New > Paste the path from step 1
+
+    ![Path variable with cmake added](./assets/path.png)
+
+    ![path ends with \bin, not \cmake.exe](./assets/path-bin.png)
+
+1. Restart your terminal
+1. Try `(Get-Command cmake).Path` in PowerShell or `which cmake` in Bash to ensure CMake can be executed from the terminal.
+
+## 2. Install Luanti dependencies
+
+Once VS and CMake are ready to use, you can install Luanti's dependencies. Luanti requires several C++ packages to function, which are installed by CMake and compiled by a C++ compiler bundled with VS. CMake uses a tool called vcpkg behind the scenes, which you may see referenced elsewhere in this project.
+
+### 2.1. Optional dependencies
+
+Some optional dependencies are recommended for advanced development. You can add them to `vcpkg.json` to install them along with the required dependencies.
+
+-   `curl[winssl]` is highly recommended, as it's required to use the server list and content store.
+-   `openal-soft`, `libvorbis` and `libogg` are optional, but required to use sound.
+-   `luajit` is optional, it replaces the integrated Lua interpreter with a faster just-in-time interpreter.
+-   `gmp` and `jsoncpp` are optional, otherwise the bundled versions will be compiled
+-   `gettext` is optional, but required to use translations.
+
+### 2.2. Install dependencies
+
+You can install dependencies via CLI or via CMake GUI, whichever your prefer.
+
+To install dependencies via CLI, you can run this program in PowerShell:
 
 ```powershell
-vcpkg install zlib zstd curl[winssl] openal-soft libvorbis libogg libjpeg-turbo sqlite3 freetype luajit gmp jsoncpp gettext[tools] opengl-registry --triplet x64-windows
+# Run this while in `path/to/luanti`
+$vs="Visual Studio 17 2022" # or "Visual Studio 16 2019", whatever matches your machine
+$toolchain_file="C:/Program Files/Microsoft Visual Studio/2022/Community/VC/vcpkg/scripts/buildsystems/vcpkg.cmake" # or wherever you installed vcpkg.cmake manually
+mkdir build
+cmake -S . -B build -G "$vs" -DCMAKE_TOOLCHAIN_FILE="$toolchain_file" -DCMAKE_BUILD_TYPE=Release -DENABLE_CURSES=OFF
```

Comment:
running a configure installs the dependencies? that is a bit confusing even if true.
any reason not to continue recommending the `vcpkg` call like before?

Reply:
The `vcpkg` call is unnecessary and overcomplicates the build process, in short.